ash/src/extensions/nv/ray_tracing.rs

```diff
@@ -0,0 +1,263 @@
+#![allow(dead_code)]
+use prelude::*;
+use std::ffi::CStr;
+use std::mem;
+use version::{DeviceV1_1, InstanceV1_1};
+use vk;
+use RawPtr;
+
+#[derive(Clone)]
+pub struct RayTracing {
+    handle: vk::Device,
+    ray_tracing_fn: vk::NvRayTracingFn,
+}
+
+impl RayTracing {
+    pub fn new<I: InstanceV1_1, D: DeviceV1_1>(instance: &I, device: &D) -> RayTracing {
```

Comment:
RTX is available for Vulkan 1.0, so this should probably be `InstanceV1_0` and `DeviceV1_0`

Reply:
Done!